### PR TITLE
Added global ms.date to docfx.json, so that API docs show a respectab…

### PR DESCRIPTION
**Pull request by natke**
Please review addition of date to these API docs. Is this the best way of doing this? Or is there an automated method?

### CODE REVIEW

File: dotnet/docfx.json [hunk 1]
```diff
@@ -56,8 +56,8 @@
       "apiPlatform": "dotnet",
       "author": "natke",
       "ms.author": "nakersha",
+      "ms.date": "11/8/2019",
       "uhfHeaderId": "MSDocsHeader-DotNet",
-      "apiPlatform": "dotnet",
```
Comment (mairaw) [inline dotnet/docfx.json]: why are you removing this? not sure what it does but it might be needed for .NET SDKs?

File: dotnet/docfx.json [hunk 2]
```diff
@@ -56,8 +56,8 @@
       "apiPlatform": "dotnet",
       "author": "natke",
       "ms.author": "nakersha",
+      "ms.date": "11/8/2019",
```
Comment (mairaw) [inline dotnet/docfx.json]:
```suggestion
      "ms.date": "11/08/2019",
```

File: dotnet/docfx.json (hunk 1 above)
Comment (natke) [reply dotnet/docfx.json]: Oh, yes, I should have noted that change too. That line was duplicated

Comment (mairaw) [reply dotnet/docfx.json]: Ah, the other one was hidden 😄

Comment (natke) [reply dotnet/docfx.json]: Yes, sorry about that

**Pull request merged**